torch/ao/quantization/_pt2e/utils.py

```diff
@@ -65,15 +77,15 @@ def _fuse_conv_bn_(m: GraphModule) -> None:
         conv_args = list(conv_op.args)
         # calling data since the fused_weight and fused_bias are nn.Parameter
         weight_attr_name = conv_args[1].target
-        setattr(m, weight_attr_name, fused_weight)
+        _set_module_attr(m, weight_attr_name, fused_weight)
```

Comment:
is this only a change for dynamo exported graph and not symbolically traced graph? can you write down the before and after in PR summary?

Reply:
It will affect not only dynamo exported graph but also a make_fx (nn_module). Sure, please check.